dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -38,6 +38,19 @@ startsecs=0
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 
+{% if not skip_chassisd and IS_MODULAR_CHASSIS == 1 %}
+[program:chassisd]
+command=python3 /usr/local/bin/chassisd
```

Comment:
It shouldn't be necessary to explicitly call `python3` here if we are only installing a Python 3 version of the daemon. Did you encounter issues without this?

Reply:
Agree, it is redundant. Removed it. Thanks.